cdap-cli/src/main/java/co/cask/cdap/cli/CLIConfig.java

```diff
@@ -77,6 +77,22 @@ public Integer get() {
     }
   };
 
+  private class AccessTokenWrapper {
```

Comment:
In general, try not to name a class "Wrapper", as it doesn't tell what it wraps. Better call it `UserAccessToken` and have a short javadoc saying what does it contain.